fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/DruidPartialDataRequestHandler.java

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web.handlers;
+
+import com.yahoo.bard.webservice.config.SystemConfig;
+import com.yahoo.bard.webservice.config.SystemConfigProvider;
+import com.yahoo.bard.webservice.druid.model.query.DruidAggregationQuery;
+import com.yahoo.bard.webservice.web.DataApiRequest;
+import com.yahoo.bard.webservice.web.responseprocessors.DruidPartialDataResponseProcessor;
+import com.yahoo.bard.webservice.web.responseprocessors.ResponseProcessor;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * A request handler that builds responses for partial data v2
+ * <p>
+ * The handler inject "uncoveredIntervalsLimit: $druid_uncovered_interval_limit" context to Druid query.
+ */
+public class DruidPartialDataRequestHandler implements DataRequestHandler {
+
+    private static final SystemConfig SYSTEM_CONFIG = SystemConfigProvider.getInstance();
+    private static final int DRUID_UNCOVERED_INTERVAL_LIMIT = SYSTEM_CONFIG.getIntProperty(
+            SYSTEM_CONFIG.getPackageVariableName("druid_uncovered_interval_limit"),
+            0
```

Comment:
This class seems to be simple enough with very simple calls. It's hard to add a test to this class with this structure. I can add test to its relevant calls if you want